packages/assets-controllers/src/NftController.test.ts

```diff
@@ -994,10 +1131,13 @@ describe('NftController', () => {
       await pendingRequest;
 
       // change the network and selectedAddress before accepting the request
+      const differentAccount = createMockInternalAccount({
+        address: '0xDifferentAddress',
```

Comment:
It's not a valid hex string, I wonder if this could cause issues here... Maybe we could use a real hex string instead?